<launch>
	<arg name="rviz" default="true" />

	<param name="voxel_size" type="double" value="2" />
	<param name="file_path" type="string" value="/home/sti/loop_data/slow" />
	<param name="pose_noise" type="double" value="0.1" />
	<node pkg="balm2" type="benchmark_realworld_TUM" name="benchmark_realworld_TUM" output="screen" />

	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find balm2)/rviz_cfg/back.rviz" />
	</group>  

</launch>
